packages/x-tailwindcss/demo/options.ts

```diff
@@ -61,12 +62,7 @@ export const options: PluginOptions = {
     return {};
   },
   theme: {
-    colors: {
-      neutral: {
-        10: 'red',
-        20: 'orange'
-      }
-    },
+    colors: {},
```

Comment:
Here you can test to override css like this:
```js
      '.button': {
        '&.outlined': {
          '&.primary': {}
        }
      }
```

@javieri-empathy if you can take a look into the type, the `&.primary` is not inferred, like it does with the `&.outlined`, despite of the `mapColors` function has the right type. I think is because the `DeepPartial` in the `ComponentsDefinition` type, but I'm not sure